ci/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM centos:centos7
```

Comment:
Is this the same file previously in ci/rpm7/Dockerfile? In that case, I'd suggest to use the old place (to minimize changes in the PR that are more costly to review)

Reply:
Not the same,  rpm7 removed, because scenarios adopted to work with any platform and version